webview-ui/src/components/chat/ChatView.tsx

```diff
@@ -733,7 +735,7 @@ const ChatView = ({ isHidden, showAnnouncement, hideAnnouncement, showHistoryVie
 						flexDirection: "column",
 						paddingBottom: "10px",
 					}}>
-					{showAnnouncement && <Announcement version={version} hideAnnouncement={hideAnnouncement} />}
+					<Announcement version={version} hideAnnouncement={hideAnnouncement} />
```

Comment:
@brownrw8 
This might be a temporary fix left over for your debugging.

```suggestion
					{showAnnouncement && <Announcement version={version} hideAnnouncement={hideAnnouncement} />}
```